### PR TITLE
ADEN-9234 Added catlapse functionality

### PR DESCRIPTION
**Pull request by preludek**

### CODE REVIEW

File: extensions/wikia/AdEngine3/src/tracking/tracker.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ export const track = (data) => {
 };
 
 export const registerSlotTracker = () => {
+	slotTracker.onChangeStatusToTrack.push('catlapsed');
```
Comment (nandy-andy) [inline extensions/wikia/AdEngine3/src/tracking/tracker.js:22]: Every-time we register slot tracker we push `'catlapsed'` -- it doesn't sound right. `registerSlotTracker` should just register the `slotTracker` it shouldn't extend the statuses to track and modify `slotTracker`. What stops us from just modifying `slotTracker` in the AdEngine?

File: extensions/wikia/AdEngine3/src/bill-the-lizard-wrapper.js [hunk 2]
```diff
@@ -6,13 +6,16 @@ import {
     context,
     events,
     eventService,
+    slotService,
     utils
 } from '@wikia/ad-engine';
 import targeting from './targeting';
 
+const AD_SLOT_CATLAPSED_STATUS = 'catlapsed';
```
Comment (nandy-andy) [inline extensions/wikia/AdEngine3/src/bill-the-lizard-wrapper.js:14]: Can we just add this to `AdSlot` in AdEngine and re-use it here like we did for `AdSlot.SLOT_RENDERED_EVENT`?

File: extensions/wikia/AdEngine3/src/tracking/tracker.js (hunk 1 above)
Comment (preludek) [reply extensions/wikia/AdEngine3/src/tracking/tracker.js:22]: Are we sure we want to move that to AdEngine? This is kind of an edge case within app and m-w. Those are the only platforms that use that and i'm not sure if we want to use that in other platforms.

File: extensions/wikia/AdEngine3/src/bill-the-lizard-wrapper.js (hunk 2 above)
Comment (preludek) [reply extensions/wikia/AdEngine3/src/bill-the-lizard-wrapper.js:14]: done

**Review by nandy-andy — APPROVED** [review]

**Pull request merged**